Description: fix bug 784912
Author: A. Maitland Bottoms <bottoms@debian.org>

# librtlsdr.pc should declare -lusb-1.0 in Libs.private section
# to exclude usb library from dynamic linking.
# References to libusb headers are not needed in Cflags, since these headers
# are not used by external rtlsdr API, but this is optional.
# https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=784912
--- a/librtlsdr.pc.in
+++ b/librtlsdr.pc.in
@@ -6,6 +6,6 @@
 Name: RTL-SDR Library
 Description: C Utility Library
 Version: @VERSION@
-Cflags: -I${includedir}/ @RTLSDR_PC_CFLAGS@
-Libs: -L${libdir} -lrtlsdr -lusb-1.0
-Libs.private: @RTLSDR_PC_LIBS@
+Cflags: -I${includedir}
+Libs: -L${libdir} -lrtlsdr
+Libs.private:  -lusb-1.0 @RTLSDR_PC_LIBS@
